src/ingress/models.py

```diff
@@ -15,7 +15,7 @@ class IngressQueue(models.Model):
 class FailedIngressQueue(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
-    endpoint = models.ForeignKey('Endpoint', on_delete=models.CASCADE)
+    endpoint = models.ForeignKey('Endpoint', on_delete=models.PROTECT)
```

Comment:
EDIT: nevermind, dit is gefixed in een latere commit.

~Dit moet ook in regel 7, in de IngressQueue (die staat nu op CASCADE).~